R/cube-index-table.R

```diff
@@ -0,0 +1,50 @@
+#' Calculate an index table for a CrunchCube
+#'
+#' Index tables are percentages of percentages. They take the percentage from
+#' `prop.table()` and divide that by the proportions of the other margin.
+#'
+#' `index.table()` is only implemented for 2 dimensional cubes. If you need to
+#' calculate indexes for a higher dimension Cube, please slice the cube first.
+#'
+#' @param x A CrunchCube to calculate index table for
+#' @param margin which margin to index against (1 for rows, 2 for columns)
+#'
+#' @return an array of percentages indexed to the margin provided
```

Comment:
But they appear not to be percentages. Should the function return `100 * out` instead?

Reply:
We can do that. Or change the docs to an array of proportions (then it matches the order of magnitude of `prop.table` which may or may not be expected or necessary). I'm ok doing either, so I'll add the `100*` unless there are strong objections.

Reply:
The only application of this that I've seen is multiplied by 100 and rounded. I'd say `*100` but not round since you can't back out the rounding if you didn't want it but you can `/ 100` if you want proportions. But I don't feel strongly and it's not like we're getting high demand for this.